src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -2,224 +2,265 @@ import React from 'react'
+                          width: calc(100% - ${!compact ? 5 * GU : 0}px);
+                        `}
+                      >
+                        {instances.map(({ proxyAddress }) => (
+                          <LocalIdentityBadge
+                            entity={proxyAddress}
+                            key={proxyAddress}
+                          />
+                        ))}
+                      </div>
+                    </DetailsGroup>
+                    <DetailsGroup>
+                      <Heading2 theme={theme}>Source code</Heading2>
+                      <BreakLink compact={compact}>
+                        {sourceUrl ? (
+                          <Link target="_blank" href={sourceUrl}>
+                            {parseHub(sourceUrl)}
+                          </Link>
+                        ) : (
+                          'No source code link.'
+                        )}
+                      </BreakLink>
+                    </DetailsGroup>
+                    {!!repoAddress && !!repoAddress && (
+                      <DetailsGroup>
+                        <Heading2 theme={theme}>Package Name</Heading2>
+                        <BreakLink compact={compact}>
+                          <Link
+                            target="_blank"
+                            href={blockExplorerUrl('address', repoAddress, {
+                              networkType: network.type,
+                            })}
+                          >
+                            {repoName}
+                          </Link>
+                        </BreakLink>
+                      </DetailsGroup>
+                    )}
+                  </div>
+                </div>
+              </div>
+            </Box>
+          }
+          secondary={
+            <Box
+              heading={
+                <div
+                  css={`
+                    display: flex;
+                    justify-content: space-between;
+                    width: 100%;
+                    padding: 0 ${2 * GU}px;
+                  `}
+                >
+                  <ReposHeader theme={theme}>App version</ReposHeader>
+                  <ReposHeader theme={theme}>Release date</ReposHeader>
+                </div>
+              }
             >
               {repoVersions}
-            </div>
-          )}
-        </DetailsGroup>
-      </div>
-    </div>
-  )
-})
+            </Box>
+          }
+        />
+      </React.Fragment>
+    )
+  }
+)
 
 AppContent.propTypes = {
   repo: RepoType.isRequired,
   repoVersions: PropTypes.node,
   onRequestUpgrade: PropTypes.func,
+  onClose: PropTypes.func,
 }
 
-const StyledLink = styled(SafeLink).attrs({ target: '_blank' })`
-  text-decoration: none;
-  color: ${theme.accent};
-
-  &:hover {
-    text-decoration: underline;
-  }
+const ReposHeader = styled.span`
+  color: ${({ theme }) => theme.contentSecondary};
+  ${textStyle('label2')};
```

Comment:
The designs have these headers with 400 font weight, rather than 600

Reply:
I think we need to a) review the designs or b) include a new `textStyle` for such cases
cc @dizzypaty @owisixseven

Reply:
After reviewing this we have decided to change `Label 2` to use 400 font weight everywhere. We will be updating designs as we go. cc @bpierre could you update this text style on aragonUI?

Reply:
Update: We won't be doing this change for now. It's postponed until we release 0.8 and review all the other text styles as well.